zeppelin-web/app/scripts/app.js

```diff
@@ -95,9 +90,11 @@ angular
     'puElasticInput',
     'xeditable'
   ])
-  .filter('breakFilter', function() {
+  .filter('breakFilter', function () {
```

Comment:
This talk is counter-productive, i've added some value that might help moving Zeppelin forward. If such contributions are not welcomed let's close this PR